packages/report-e2e-tests/src/examples/combined-results-with-issues.snap.html

```diff
@@ -556,6 +556,55 @@
           margin: 24px 16px 0 16px;
         }
       }
+      .scan-details-section {
```

Comment:
These (and below with the congrats message styles) are added as a result of using automated-checks-report.scss which has two import statements that are not utilized in summary report.